label_studio/tasks/models.py

```diff
@@ -200,77 +200,13 @@ def storage(self):
             from io_storages.s3.models import S3ImportStorage
             return S3ImportStorage()
 
-    def update_is_labeled(self):
-        """Set is_labeled field according to annotations*.count > overlap
-        """
-        n = self.annotations.filter(Q_finished_annotations & Q(ground_truth=False)).count()
-        # self.is_labeled = n >= self.project.maximum_annotations
-        self.is_labeled = n >= self.overlap
-
-    def reset_updates(self):
-        """ Reset updates to default from model for one task.
-            We need it in duplicate project or total deletion of annotations
-        """
-        for field in Task._meta.fields:
-            if field.name in Task.updates:
-                setattr(self, field.name, field.default)
-
-    @staticmethod
-    def bulk_reset_updates(project):
-        """ Bulk reset updates to default, it's a fast way to reset all tasks in project
-        """
-        for field in Task._meta.fields:
-            if field.name in Task.updates:
-                project.tasks.update(**{field.name: field.default})
-
-    @staticmethod
-    def bulk_update_is_labeled(project):
```

Comment:
I'm not sure it's a good idea to remove it. Yes, it's not used anywhere, but it must be used in all bulk imports for tasks and now it's a bug.